src/components/Fcrackzip/Fcrackzip.tsx

```diff
@@ -1,19 +1,29 @@
-import { Button, LoadingOverlay, Stack, TextInput, NativeSelect, Checkbox } from "@mantine/core";
+import { Button, Stack, TextInput, NativeSelect, Checkbox } from "@mantine/core";
 import { useForm } from "@mantine/form";
 import { useEffect, useCallback, useState } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
-import { UserGuide } from "../UserGuide/UserGuide";
+import { RenderComponent } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile } from "../SaveOutputToFile/SaveOutputToTextFile";
```

Comment:
As part of baselining, this should be changed to aveOutputToTextFile_v2. 
```suggestion
import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
```